docs/production-setup.md

```diff
@@ -217,34 +288,60 @@ In order to be able to propose batches we are going to register our Ethereum acc
 to do this execute this command:
 
 ```bash
-docker run --rm -v /$HOME/zkevm-node/config.toml:/app/config.toml hermeznetwork/zkevm-node:latest sh -c "./zkevm-node register --cfg=/app/config.toml --network=internaltestnet --y <public IP or URL for users to access the sequencer> "
+docker run --rm -v /$HOME/zkevm-node/config.toml:/app/config.toml \
+    -v /$HOME/zkevm-node/genesis.json:/app/genesis.json \
+    hermeznetwork/zkevm-node:latest \
+    sh -c "./zkevm-node register --cfg=/app/config.toml --y <public IP or URL for users to access the sequencer> "
 ```
 
 In order to propose new batches, you must approve the Tokens to be used by the Roll-up on your behalf, to do this execute this command:
 > remember to set the value of the parameter amount before executing
 
 ```bash
-docker run --rm -v /$HOME/zkevm-node/config.toml:/app/config.toml hermeznetwork/zkevm-node:latest sh -c "./zkevm-node approve --cfg=/app/config.toml --network=internaltestnet --address=poe --amount=0 --y"
+docker run --rm -v /$HOME/zkevm-node/config.toml:/app/config.toml \
+    -v /$HOME/zkevm-node/genesis.json:/app/genesis.json \
+    hermeznetwork/zkevm-node:latest \
+    sh -c "./app/zkevm-node approve --am [amount] -y --genesis /app/genesis.json --cfg /app/config.toml""
 ```
```

Comment:
even though this is necessary when you want to send sequences, this is not the case for users now, because only the trusted sequencer can send sequences at this time.

Maybe we should have different documents for different ways of running the node, `sync only`, `sequencer` and `aggregator`.

In this case, this is the `sync only` doc, theoretically, we should not have this here.

WDTY @arnaubennassar?

Reply:
Ok, removing it.

Reply:
I agree, we should have different docs for each role, but this is ok for now... the documentation is quite outdated :sweat_smile: